config/environments/development.rb

```diff
@@ -39,4 +39,6 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
+
+  config.web_console.whitelisted_ips = '192.168.59.3'
```

Comment:
@machuga I think this is actually a Docker specific change.  Noting in web-console documentation:

```
Take a note that IPv4 and IPv6 localhosts are always allowed. This wasn't the case in 2.0.
```